### DISCUSSION TITLE
Error in starting nginx forwarding

### DISCUSSION THREAD

**Discussion by zengxc-1996** [apache/couchdb#3230] [Q&A]
## Steps to Reproduce (for bugs)

1.  configure nginx :      proxy_pass  http://127.0.0.1:5984/_utils/;
2.   http://my ip/  " You are not a server admin" when you log in   
3.
4.

## Your Environment

* Version used: 3.1.1
* Browser Name and version:  google  latest
* Operating System and version (desktop or mobile): 
* Link to your project:

**Comment by aellerton** [comment]
I think ordinarily it'd be good to have more of the usual sections filled out, but as I've solved this some time ago, I have a strong indication of what's going on.

Fauxton expects certain paths to work a certain way. Specifically, `_session` needs to be proxied as well. This was found by looking at Chrome development tools.

We mount CouchDB under `/db` (though you don't have to do that), so everything that is normally under say http://127.0.0.1:5984/something is available at http://servername/db/something

It works well but proxy location `_session` to CouchDB as well.

I also found relying on the bundled version of Fauxton a bit problematic, so do a build of Fauxton from source where I set the root to "../db".

Hope this helps
